x-pack/plugins/infra/public/components/asset_details/tabs/metadata/add_metadata_filter_button.tsx

```diff
@@ -8,9 +8,9 @@
 import React, { useMemo } from 'react';
 import { i18n } from '@kbn/i18n';
 import { EuiToolTip, EuiButtonIcon } from '@elastic/eui';
-import { useMetricsDataViewContext } from '../../../pages/metrics/hosts/hooks/use_data_view';
-import { useKibanaContextForPlugin } from '../../../hooks/use_kibana';
-import { useUnifiedSearchContext } from '../../../pages/metrics/hosts/hooks/use_unified_search';
+import { useMetricsDataViewContext } from '../../../../pages/metrics/hosts/hooks/use_data_view';
+import { useKibanaContextForPlugin } from '../../../../hooks/use_kibana';
+import { useUnifiedSearchContext } from '../../../../pages/metrics/hosts/hooks/use_unified_search';
```

Comment:
The action performed by this component shouldn't be tightly coupled with `pages/*` dependencies, as it may change depending on what's consuming it.